### PR TITLE
Feat: add signup password strength validation

### PR DESCRIPTION
**Pull request by raphael-arce**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -18,7 +21,7 @@ NEXT_PUBLIC_MAP_BOUNDING_BOX=13.0824446341071,52.3281202651866,13.7682544186827,
 # See https://github.com/technologiestiftung/giessdenkiez-de-postgres-api
 NEXT_PUBLIC_SUPABASE_URL=http://localhost:54321
 NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0
-NEXT_PUBLIC_API_ENDPOINT=https://localhost:8080
+NEXT_PUBLIC_API_ENDPOINT=http://localhost:8080
```
Comment (dnsos) [inline .env.sample:24]: Thanks for that fix!

File: .env.sample [hunk 2]
```diff
@@ -3,6 +3,9 @@
 # The absolute base url
 NEXT_PUBLIC_BASE_URL=http://localhost:3000
 
+# Supabase will redirect to this url after recovery
+NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_URL=http://localhost:3000/auth
```
Comment (dnsos) [inline .env.sample:7]: I've realized we have introduced a duplication here: the `http://localhost:3000` part is always the same as `NEXT_PUBLIC_BASE_URL`.

I would suggest we change this line to `NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_PATH=/auth` and then combine it with the `NEXT_PUBLIC_BASE_URL` in the code.

**Review by dnsos — CHANGES_REQUESTED** [review]
Looks good to me, thank you @raphael-arce.

There is one issue concerning the env variables that should be handled, see my comment below.

File: .env.sample (hunk 2 above)
Comment (raphael-arce) [reply .env.sample:7]: Great idea, will fix 👍

Comment (raphael-arce) [reply .env.sample:7]: suggestion: how about we keep how it is in the code, and use this in the .env instead:
`NEXT_PUBLIC_RECOVERY_AUTH_REDIRECT_URL=${NEXT_PUBLIC_BASE_URL}auth`

what do you think?

Comment (dnsos) [reply .env.sample:7]: I'm not sure if this works in our production environment. I have a feeling that Vercel doesn't allow for one variable to reference another. But feel free to check it out!

Comment (ff6347) [reply .env.sample:7]: Next.js can do variable expansion:

https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

>Note: Next.js will automatically expand variables ($VAR) inside of your .env* files. This allows you to reference other secrets, like so:

```plain
# .env
HOSTNAME=localhost
PORT=8080
HOST=http://$HOSTNAME:$PORT

```

Comment (ff6347) [reply .env.sample]: Try the notation ${VAR_NAME}. It should be valid, at least it is on posix shells.

**Review by dnsos — APPROVED** [review]

**Pull request merged**